spec/integration/connection_spec.rb

```diff
@@ -16,28 +16,84 @@
   end
   after(:all) { Process.kill('QUIT', @pid) }
 
-  MockServer.rpc_descs.each do |rpc, desc|
-    rpc = GRPC::GenericService.underscore(rpc.to_s)
+  describe 'WorkflowService' do
+    let(:service) { subject.workflow_service }
 
-    # TODO: Remove once https://github.com/temporalio/sdk-core/issues/335 fixed
-    next if rpc == 'get_workflow_execution_history_reverse'
+    MockWorkflowService.rpc_descs.each do |rpc, desc|
+      rpc = GRPC::GenericService.underscore(rpc.to_s)
 
-    describe "##{rpc}" do
+      # TODO: Remove once https://github.com/temporalio/sdk-core/issues/335 fixed
+      next if rpc == 'get_workflow_execution_history_reverse'
```

Comment:
Linked issue's been fixed for quite some time

Reply:
Ah, it was probably still open when I first created that file, I'll remove it, thanks for pointing out 👍